config/jobs/kubernetes/kops/kops-presubmits.yaml

```diff
@@ -104,6 +114,7 @@ presubmits:
       testgrid-dashboards: sig-cluster-lifecycle-kops, presubmits-kops, kops-presubmits
       testgrid-tab-name: e2e-do-gossip
   - name: pull-kops-e2e-kubernetes-do-fqdn
+    cluster: eks-prow-build-cluste
```

Comment:
I don't think we have `preset-do-credential` on the new cluster (?)

Also: typo in cluster name.

Suggest doing this one in a different PR.